src/main/java/seedu/address/model/pdf/Deadline.java

```diff
@@ -6,26 +6,29 @@
 import java.time.LocalDate;
 
 /**
- * Represents a Pdf's address in the address book.
+ * Represents a Pdf's deadline in the Pdf book.
  * Guarantees: immutable;
  */
-public class Deadline {
+public class Deadline implements Comparable<Deadline> {
     public static final String MESSAGE_CONSTRAINTS = "Deadline can take any valid date, "
             + "and it should not be blank";
+    public static final int STATUS_READY = 0;
+    public static final int STATUS_COMPLETE = 1;
+    public static final int STATUS_REMOVE = 2;
```

Comment:
For the above 3 values I'd suggest using an internal `Enum` Class to store the value instead of an explicit `STATUS_TYPE = VALUE`